### PR TITLE
feat: add environment variables to change fzf keys

### PR DESCRIPTION
**Pull request by benelan**
Add `GH_NOTIFY_<ACTION>_KEY` environment variables so the fzf keys can be changed. The extension remaps some of fzf's defaults (e.g., `ctrl-p`) and my muscle memory was making it hard to use. Let me know if you'd like any name or code changes, thanks!

### CODE REVIEW

**Comment by LangLangBart** [comment]
Thanks for the PR.

I would suggest changing the assignment to use the following colon (`:`) syntax[^1]:

```sh
: "${ENV_VAR:=default}"
```

instead of:

```sh
ENV_VAR="${ENV_VAR:=default}"
```

This change avoids writing the environment variable name twice during the assignment. Exporting the variable is not needed, as it will not be used in any child processes.

```sh
# Customize the fzf keys using environment variables
: "${GH_NOTIFY_MARK_ALL_READ_KEY:=ctrl-a}"
: "${GH_NOTIFY_OPEN_BROWSER_KEY:=ctrl-b}"
: "${GH_NOTIFY_VIEW_DIFF_KEY:=ctrl-d}"
: "${GH_NOTIFY_VIEW_PATCH_KEY:=ctrl-p}"
: "${GH_NOTIFY_RELOAD_KEY:=ctrl-r}"
: "${GH_NOTIFY_MARK_READ_KEY:=ctrl-t}"
: "${GH_NOTIFY_COMMENT_KEY:=ctrl-x}"
: "${GH_NOTIFY_RESIZE_PREVIEW_KEY:=btab}"
: "${GH_NOTIFY_VIEW_KEY:=enter}"
: "${GH_NOTIFY_TOGGLE_PREVIEW_KEY:=tab}"
: "${GH_NOTIFY_TOGGLE_HELP_KEY:=?}"
```

---

It would probably be good to include a concise example in the `README` for reassigning a key:

````md
To avoid conflicting with user keybindings, redefine them, e.g., change `ctrl-p` to `ctrl-u`.

```sh
GH_NOTIFY_VIEW_PATCH_KEY="ctrl-u" gh notify
```

**NOTE:** The assigned key must be a valid key listed under `AVAILABLE KEYS` in the `fzf` man page.
````


[^1]: [Bash Reference Manual - Shell Parameter Expansion](https://www.gnu.org/software/bash/manual/bash.html#Shell-Parameter-Expansion)

File: readme.md [hunk 1]
```diff
@@ -48,21 +48,21 @@ gh notify [Flags]
...
-| <kbd>ctrl</kbd><kbd>b</kbd>    | browser                                             |
-| <kbd>ctrl</kbd><kbd>d</kbd>    | view diff                                           |
-| <kbd>ctrl</kbd><kbd>p</kbd>    | view diff in patch format                           |
-| <kbd>ctrl</kbd><kbd>r</kbd>    | reload                                              |
-| <kbd>ctrl</kbd><kbd>t</kbd>    | mark the selected notification as read and reload   |
-| <kbd>ctrl</kbd><kbd>x</kbd>    | write a comment with the editor and quit            |
-| <kbd>esc</kbd>                 | quit                                                |
+| Keys                           | Description                                         | Customization Environment Variable      |
+| ------------------------------ | --------------------------------------------------- | --------------------------------------- |
+| <kbd>?</kbd>                   | toggle help                                         | <kbd>GH_NOTIFY_TOGGLE_HELP_KEY</kbd>    |
```
Comment (LangLangBart) [inline readme.md]: `kbd` is more typical for keybinds, the variable can just be wrapped in backticks
```suggestion
| <kbd>?</kbd>                   | toggle help                                         | `GH_NOTIFY_TOGGLE_HELP_KEY`    |
```

**Comment by benelan** [comment]
Thanks for the review! I made the suggested changes, let me know if anything else is needed on my end.

**Comment by LangLangBart** [comment]
Thanks, GitHub seems to be having some trouble executing the workflow run.

https://github.com/meiji163/gh-notify/actions/runs/9952278435

I will push a small change to see if the issue is specific to a user or happens in general.

**Comment by benelan** [comment]
Not sure if it's related, but in the past I've resolved GitHub Action problems for pull requests from forks using the [`pull_request_target`](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target) event instead.

**Comment by LangLangBart** [comment]
> pull_request_target

Can you make the change and push it to see if this might resolve the issue?

**Comment by benelan** [comment]
You may need to [manually run the workflows](https://docs.github.com/en/actions/managing-workflow-runs/approving-workflow-runs-from-public-forks#approving-workflow-runs-on-a-pull-request-from-a-public-fork) since it is my first time contributing

**Comment by LangLangBart** [comment]
> You may need to [manually run the workflows](https://docs.github.com/en/actions/managing-workflow-runs/approving-workflow-runs-from-public-forks#approving-workflow-runs-on-a-pull-request-from-a-public-fork) since it is my first time contributing

I  approved that earlier and it should run, maybe one has to apply it to the main branch before taking effect. I will make commit to main branch with `pull_request_target`.

**Comment by benelan** [comment]
looks like that fixed it!

**Comment by LangLangBart** [comment]
> looks like that fixed it!

Thanks for helping out. I will use it a bit more and merge it in one hour.

**Comment by LangLangBart** [comment]
@benelan Thanks for the PR. You were right to export it initially. I forgot about the help page (`?`) inside `fzf`.

**Pull request merged**